core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorShimServiceSpec.js

```diff
@@ -138,6 +144,12 @@ describe('Simple Editor Shim Service', function() {
     expect(html).toEqual(states[givenState].content[0].value);
   });
 
+  it('should return interaction id for given state', function() {
+    var givenState = currentStateName;
```

Comment:
The naming isn't great here: don't confuse a state with the state name. I think you can just use the currentStateName variable directly throughout.